### ISSUE TITLE
Not able to search the album by distributor or LICENSOR NAME	

### ISSUE THREAD

**Issue by amrutan** [spotify/web-api#1304]
Title: Not able to search the album by distributor or LICENSOR NAME	

#### Endpoint(s):
* `GET /search/v1`

#### Scope(s):
Not able to search the album by distributor or LICENSOR NAME.
We want to search the album which belongs to specific distributor only. How we can do that?

**Comment by daviddassau** [comment]
Yes! This would be super handy. I work for a music label, and I'm about to start working on an application that will use the Spotify API. Being able to query using the Licensor Name would be amazing!
